osgar/drivers/sicklidar.py

```diff
@@ -37,7 +39,26 @@ def parse_raw_data(raw_data):
         scan_start = 26
         scan_end = scan_start + scan_size
         dist = [int(x, 16) for x in data[scan_start:scan_end]]
-        return dist
+
+        remission = None
+        if scan_end + 100 < len(data):  # TODO properly detect presence RSSI
+            # should have remissing
+            rssi_index = scan_end + 1
+            assert data[rssi_index] == b'RSSI1', data[rssi_index]
+#            resolution2 = int(data[rssi_index + 1], 16)
+#            assert resolution == resolution2, (resolution, resolution2)
+            assert int(data[rssi_index + 2], 16) == 0, data[rssi_index + 2]
+#            start_angle = int(data[rssi_index + 3], 16)  # unsigned HEX
+#            assert start_angle == -1350, start_angle
+            angular_step = int(data[rssi_index + 4], 16)
+            assert angular_step == resolution, (angular_step, resolution)
+
+            amount = int(data[rssi_index + 5], 16)
+            assert amount == scan_size, (amount, scan_size)
+            rssi_index += 6
+            remission = [int(x,16) for x in data[rssi_index:rssi_index + amount]]
```

Comment:
Does this parse all data left or is there anything more? I mean, would this hold?
```python
assert rssi_index + amount == len(data)
```

Reply:
If you mean RSSI data then yes, that should be all. But the protocol is variable and contain for example encoders or other sections

Reply:
So the assert wouldn't hold. Ok, thanks.